src/Commands/Make/Base.js

```diff
@@ -68,7 +68,8 @@ class MakeBase extends Command {
     const templateFile = path.join(__dirname, '../../Generators/templates', `${templateFor}.mustache`)
 
     const filePath = generators[templateFor].getFilePath(name, options)
-    const data = generators[templateFor].getData(name, flags)
+    const baseName = path.basename(name)
+    const data = generators[templateFor].getData(baseName, flags)
```

Comment:
You can please just inline the `path.basename(name)` inside the `getData` call.

Example

```js
generators[templateFor].getData(path.basename(name), flags)
```